Mathlib/Topology/EMetricSpace/Defs.lean

```diff
@@ -59,21 +59,21 @@ def uniformSpaceOfEDist (edist : α → α → ℝ≥0∞) (edist_self : ∀ x :
     ⟨ε / 2, ENNReal.half_pos ε0.ne', fun _ h₁ _ h₂ =>
       (ENNReal.add_lt_add h₁ h₂).trans_eq (ENNReal.add_halves _)⟩
 
--- the uniform structure is embedded in the emetric space structure
--- to avoid instance diamond issues. See Note [forgetful inheritance].
-/-- Extended (pseudo) metric spaces, with an extended distance `edist` possibly taking the
-value ∞
-
-Each pseudo_emetric space induces a canonical `UniformSpace` and hence a canonical
-`TopologicalSpace`.
-This is enforced in the type class definition, by extending the `UniformSpace` structure. When
-instantiating a `PseudoEMetricSpace` structure, the uniformity fields are not necessary, they
-will be filled in by default. There is a default value for the uniformity, that can be substituted
-in cases of interest, for instance when instantiating a `PseudoEMetricSpace` structure
-on a product.
-
-Continuity of `edist` is proved in `Topology.Instances.ENNReal`
--/
+/-- A pseudo extended metric space is a type endowed with a `ℝ≥0∞`-valued distance `edist`
+satisfying reflexivity `edist x x = 0`, commutativity `edist x y = edist y x`, and the triangle
+inequality `edist x z ≤ edist x y + edist y z`.
+
+Note that we do not require `edist x y = 0 → x = y`. See `EMetricSpace` for the analogous class with
+that stronger assumption.
+
+Any pseudo extended metric space is a topological space and a uniform space,
+where the topology and uniformity come from the metric.
+Note that a T1 pseudo extended metric space is just an extended metric space.
```

Comment:
```suggestion
Note that a T1 pseudo extended metric space is just an extended metric space (`EMetricSpace`).
```
Similarly above, it could be nice to reference the other classes in a linkable way.

Reply:
But the paragraph just above doesn't make the connection that `EMetricSpace` is the same thing as extended metric space: that's what I want added!

Reply:
Do you also want to mention TopologicalSpace, UniformSpace, T1 in a similar way? In particular, since the next paragraph discusses things which are related to the implementation of those classes